sleap/gui/app.py

```diff
@@ -1236,6 +1236,12 @@ def _has_topic(topic_list):
         if _has_topic([UpdateTopic.frame, UpdateTopic.project_instances]):
             self.state["last_interacted_frame"] = self.state["labeled_frame"]
 
+        if _has_topic([UpdateTopic.sessions]):
+            self.sessions_dock.table.model().items = self.labels.videos
```

Comment:
I think we may need to specify which table in the list of tables here, perhaps `self.sessions_dock.cameras_table`  (see what is in `self.sessions_dock.table` below).
https://github.com/talmolab/sleap/blob/7f74a8ba75aa2bd9d7cc53a156fcc2efa188607a/sleap/gui/widgets/docks.py#L70
https://github.com/talmolab/sleap/blob/7f74a8ba75aa2bd9d7cc53a156fcc2efa188607a/sleap/gui/widgets/docks.py#L625-L636

I received the following error message with the current implementation:
```
Happy SLEAPing! :)
Restoring GUI state...
Traceback (most recent call last):
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 290, in openProject
    self.execute(OpenProject, filename=filename, first_open=first_open)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 244, in execute
    command().execute(context=self, params=kwargs)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 140, in execute
    self.do_with_signal(context, params)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 164, in do_with_signal
    cls.do_action(context, params)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 759, in do_action
    new_window.commands.loadProjectFile(filename)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 275, in loadProjectFile
    self.execute(LoadProjectFile, filename=filename)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 244, in execute
    command().execute(context=self, params=kwargs)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 140, in execute
    self.do_with_signal(context, params)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 164, in do_with_signal
    cls.do_action(context, params)
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\commands.py", line 710, in do_action
    context.app.on_data_update([UpdateTopic.project, UpdateTopic.all])
  File "d:\social-leap-estimates-animal-poses\source\sleap\sleap\gui\app.py", line 1240, in on_data_update
    self.sessions_dock.table.model().items = self.labels.videos
AttributeError: 'list' object has no attribute 'model'
```